velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -30,6 +30,11 @@ inline void registerArraySumFunction() {
   registerFunction<ArraySumFunction, OT, IT>({"array_sum_alt"});
 }
 
+template <typename OT, typename IT>
```

Comment:
Nit: Since we don't bind this function for multiple types, the usage of template variables doesn't seem worthwhile to me.

Reply:
+1

Reply:
I have removed the registerArrayAverageFunction() method.